recipes/Data-Prep-Kit/intro/dpk_intro_1_python.ipynb

```diff
@@ -0,0 +1,1123 @@
+        "STAGE = 1\n",
+        "\n",
+        "input_folder = MY_CONFIG.INPUT_DATA_DIR\n",
+        "output_folder =  output_parquet_dir\n",
+        "\n",
+        "print (f\"🏃🏼 STAGE-{STAGE}: Processing input='{input_folder}' --> output='{output_folder}'\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "9bb15f02-ab5c-4525-a536-cfa1fd2ba70b",
+      "metadata": {
+        "id": "9bb15f02-ab5c-4525-a536-cfa1fd2ba70b"
+      },
+      "source": [
+        "### 3.2 - Execute"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "b0cd8ebd-bf71-42d6-a397-8df0c7b66a26",
+      "metadata": {
+        "id": "b0cd8ebd-bf71-42d6-a397-8df0c7b66a26"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "\n",
+        "import ast\n",
+        "import os\n",
+        "import sys\n",
+        "\n",
+        "from pdf2parquet_transform import (\n",
+        "    pdf2parquet_contents_type_cli_param,\n",
+        "    pdf2parquet_contents_types,\n",
+        ")\n",
+        "from data_processing.runtime.pure_python import PythonTransformLauncher\n",
+        "from pdf2parquet_transform_python import Pdf2ParquetPythonTransformConfiguration\n",
+        "\n",
+        "from data_processing.utils import GB, ParamsUtils\n",
+        "\n",
+        "\n",
+        "# create parameters\n",
+        "local_conf = {\n",
+        "    \"input_folder\": input_folder,\n",
+        "    \"output_folder\": output_folder,\n",
+        "}\n",
+        "ingest_config = {\n",
+        "    pdf2parquet_contents_type_cli_param: pdf2parquet_contents_types.JSON,\n",
+        "}\n",
+        "\n",
+        "params = {\n",
+        "    # Data access. Only required parameters are specified\n",
+        "    \"data_local_config\": ParamsUtils.convert_to_ast(local_conf),\n",
+        "    \"data_files_to_use\": ast.literal_eval(\"['.pdf']\"),\n",
+        "}\n",
+        "\n",
+        "\n",
+        "sys.argv = ParamsUtils.dict_to_req(d=(params | ingest_config))\n",
+        "# create launcher\n",
+        "launcher = PythonTransformLauncher(Pdf2ParquetPythonTransformConfiguration())\n",
+        "# launch\n",
+        "return_code = launcher.launch()\n",
+        "\n",
+        "if return_code == 0:\n",
+        "    print (f\"✅ Stage:{STAGE} completed successfully\")\n",
+        "else:\n",
+        "    raise Exception (\"❌ Job failed\")\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "5ca790e0",
+      "metadata": {
+        "id": "5ca790e0"
+      },
+      "source": [
+        "### 3.3 - Inspect Generated output\n",
+        "\n",
+        "Here we should see one entry per input file processed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "fe59563d",
+      "metadata": {
+        "id": "fe59563d"
+      },
+      "outputs": [],
+      "source": [
+        "from utils import read_parquet_files_as_df\n",
```

Comment:
The user won't notice that this `utils` is actually the `utils.py` file, which has useful code they might want to reuse. Add to your previous cell a comment that they should look at this file to see what it is doing. (I assume the same file is used in the other notebook, so I would add the same comment there.)